server/channels/web/webhook.go

```diff
@@ -63,12 +59,13 @@ func incomingWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 		}
 	}()
 
+	errCtx["media_type"] = mediaType
 	if mediaType == "application/x-www-form-urlencoded" {
 		payload := strings.NewReader(r.FormValue("payload"))
 
-		incomingWebhookPayload, err = decodePayload(payload)
-		if err != nil {
-			c.Err = err
+		incomingWebhookPayload, appErr = decodePayload(payload)
+		if appErr != nil {
+			c.Err = model.NewAppError("incomingWebhook", "web.incoming_webhook.decode.app_error", errCtx, "", http.StatusBadRequest).Wrap(appErr)
```

Comment:
Are we returning the wrapped error via the API? If not, is that something we want to keep?

Reply:
Should the error message make it clear that the logs need to be checked?

Reply:
- Yes, wrapped error does show in the API response. So, we should keep it.
- Error messages telling to check error logs, seems a bit too much to me.